packages/ceramic-core/src/ceramic.ts

```diff
@@ -66,7 +67,11 @@ class Ceramic {
   }
 
   static async create(ipfs: Ipfs.Ipfs, config: CeramicConfig = {}): Promise<Ceramic> {
-    const dispatcher = new Dispatcher(ipfs)
+    const pinningServiceFactory = new PinningServiceFactory(ipfs, config)
+    const pinningService = pinningServiceFactory.get()
+
+    const dispatcher = new Dispatcher(ipfs, pinningService)
```

Comment:
Why do you pass the pinningService into the dispatcher? Any reason to bundle it like this?